### PR TITLE
Properly discover filename from Python Scripts

### PR DESCRIPTION
**Pull request by lresende**
Fixes https://github.com/elyra-ai/elyra/issues/1050
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: kfp_notebook/pipeline/_notebook_op.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def __init__(self,
         """
 
         self.notebook = notebook
-        self.notebook_name = self._get_file_name_with_extension(notebook, 'ipynb')
+        self.notebook_name = self._get_file_name_with_extension(notebook)
```
Comment (kevin-bates) [inline kfp_notebook/pipeline/_notebook_op.py:74]: If all the method is doing is call basename, we should just cut to the chase.
```suggestion
        self.notebook_name = os.path.basename(notebook)
```

Comment (kevin-bates) [reply kfp_notebook/pipeline/_notebook_op.py:74]: However, what if there is no extension here?  Should this be the place where we enforce an extension of either `.ipynb` or `.py`?  In that case, the method would still be useful to house that raising of exception.

**Review by kevin-bates — COMMENTED** [review]
Had a couple comments.  Seems like we should ensure the proper extension is present.

File: kfp_notebook/pipeline/_notebook_op.py (hunk 1 above)
Comment (lresende) [reply kfp_notebook/pipeline/_notebook_op.py]: the notebook in this case is the path of the file and must be valid. In this case, we might tight the validation but I don't believe we should care about the missing extension.

**Pull request merged**